.github/ISSUE_TEMPLATE/hotfix-issue-release-template.yml

```
@@ -0,0 +1,172 @@
+name: 🔥 HotFix
+description: Create a hotfix release issue for a project (Project maintainers only).
```

Refactor to below.  Do this for all the templates

```suggestion
description: Create a hotfix release issue for the project (Project maintainers only).
```